testsuite/features/secondary/trad_action_chain.feature

```diff
@@ -27,9 +27,10 @@ Feature: Action chain on traditional clients
     And I wait until the table contains "FINISHED" or "SKIPPED" followed by "FINISHED" in its first rows
 
   Scenario: Pre-requisite: remove all action chains before testing on traditional client
-    Given I am logged in via XML-RPC actionchain as user "admin" and password "admin"
+    Given I am logged in API as user "admin" and password "admin"
     When I delete all action chains
     And I cancel all scheduled actions
+    And I logout from API
```

Comment:
Same, we are adding additional logouts that are changing the behavior the tests had before this PR.
And they might not even bring any good.